### ISSUE TITLE
check user input config exist or not

### ISSUE THREAD

**Issue by shenxianpeng** [commit-check/commit-check#302]
### describe your issue

```bash
sxp@pc ~/c/commit-check (main)> commit-check --config my-config.toml -m
sxp@pc ~/c/commit-check (main)> echo $status
0
```

If the user runs the above command, it should return failed and explain why.

But right now it does not show an error, and the return code is still 0

### commit-check.toml or cchk.toml

```toml

```
